src/content/guides/loading-and-sending-assets.mdx

```diff
@@ -10,6 +10,34 @@ import CodePanel from '../../components/CodePanel.astro';
 
 [Assets](/api/assets) such as Tokens can be issued by a Centrapay Account. They can then be sent to another Centrapay account using one of the methods described below.
 
+## Requesting Assets
+
+You can request for Assets such as Tokens and Giftcards by calling our [Request Asset Transfers](/api/asset-transfers/#request-an-asset-transfer) endpoint.
+
+For any requests for an asset transfer that is not claimed within 2 weeks, the requested asset transfer will expired.
+
+Asset Transfer Request goes through different lifecycle stages
```

Comment:
This is wrong.
1. You cannot request giftcard or tokens only quartz asset
2. Asset transfer requests don't get claimed rather they get accepted.